client/src/browse/maps/MapMarker.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useContext } from "react";
+import styled from "styled-components";
+
+import MarkerCard from "./MarkerCard.js";
+import { RiMapPin2Fill } from "react-icons/ri";
+// import {FaMapMarkerAlt } from "react-icons/fa";
+const MapMarker = (props) => {
```

Comment:
Remove commented out lines unless you will need them later

Reply:
Comment removed